components/SearchSandbox/components/Header.js

```diff
@@ -155,7 +155,12 @@ export default class Header extends Component {
 				style={{ display: 'flex', padding: '10px 20px 0', justifyContent: 'space-between' }}
 			>
 				<div style={{ marginTop: 8, marginRight: 8 }}>
-					<Switch checked={analytics} onChange={toggleAnalytics} /> Enable analytics
+					<Tooltip
+						placement="right"
+						title="Switch for whether to record analytics based on the search and click requests made from the Search Preview view. It's enabled by default."
+					>
+						<Switch checked={analytics} onChange={toggleAnalytics} /> Record Analytics
```

Comment:
Just a quick question: If we click Record Analytics it wont switch the values as I dont see any label or ids attached. Am I right ?